src/DetailsView/reports/components/report-sections/instance-details.tsx

```diff
@@ -22,12 +23,55 @@ export const InstanceDetails = NamedSFC<InstanceDetailsProps>('InstanceDetail',
             </tr>
         );
     };
+
+    const renderInstructions = (howToFixString: string) => {
```

Comment:
2 questions about the how to fix instructions code:
1. Can we move this to a separated component, it's implementation details the `InstanceDetails` component doesn't need to know
2. Any reason why we are implementing this from scratch instead of reusing `FixInstructionPanel` (src\injected\components\fix-instruction-panel.tsx)